tensortrade/oms/services/execution/simulated.py

```diff
@@ -112,11 +115,14 @@ def execute_sell_order(order: 'Order',
     commission = options.commission * filled
     quantity = filled - commission
 
-    if commission.size < Decimal(10) ** -quantity.instrument.precision:
-        logging.warning("Commission is less than instrument precision. Canceling order. "
-                        "Consider defining a custom instrument with a higher precision.")
-        order.cancel("COMMISSION IS LESS THAN PRECISION.")
-        return None
+    precision_to_have = Decimal(10) ** -quantity.instrument.precision
+
+    if commission.size > 0.0:
```

Comment:
No more logging ? Is it ok without?

Reply:
Yes. This enables 0% fees and fixes what would've been a crash when fees are below asset precision. If the fees are equal to 0%, then no commission is added to the calculations (it would've crashed before this update). If the fees are above 0% and over asset precision, the old behavior is the same than before the update. The last case is if fees are above 0% but below asset precision, this too would've crashed before this update. Instead of crashing with logs in this case, I update the fees to equal asset precision. Nothing complicated here. Just a nested if block testing for those cases and making the necessary changes.